suleyunus/itv-app
# Project setup

### What does this PR do?

Set ups the project to use express and sequalize

### Description of task to be completed?

Have a express server running

### How can this be tested manually?

After cloning this repo, cd into it and run:
```
npm install
```
* After the dependencies have installed, run `npm run start:dev` to fire up the server
* Visit local host at the port shown and you should the message "hello world" in json.
